app/templates/components/forms/admin/settings/billing.hbs

```diff
@@ -88,5 +88,5 @@
     <label for="adminBillingAdditionalInfo">{{t 'Additional Info'}}</label>
     <Input @type="text" @id="adminBillingAdditionalInfo" @value={{this.model.adminBillingAdditionalInfo}} />
   </div>
-  <button class="ui teal button" type="submit" name="submit">{{t 'Save'}}</button>
+  <button class="ui teal button" type="button" name="submit" onclick={{action "submit"}}>{{t 'Save'}}</button>
```

Comment:
I don't think this should be needed, just type="submit" should do

Reply:
keeping it to `submit` that is triggering twice